packages/hash/api/src/graphql/resolvers/ontology/property-type.ts

```diff
@@ -27,9 +28,9 @@ export const createPropertyTypeResolver: ResolverFn<
   const createdPropertyType = await createPropertyType(
     { graphApi },
     {
-      ownedById: ownedById ?? userModel.getEntityUuid(),
+      ownedById: brand(ownedById ?? userModel.getEntityUuid()),
```

Comment:
If we change `ownedById` to `OwnedById` then I guess we could have
```suggestion
      ownedById: brand(ownedById) ?? userModel.getEntityUuid(),
```

and ideally `userModel.getEntityUuid()` will change to a nicer standalone function anyway like `extractAccountIdFromUser` or something

Reply:
Good idea! da18df6a18a02639b03d7c83bc9867551d087b09